tool/teleport/common/teleport.go

```diff
@@ -549,11 +554,36 @@ func onConfigDump(flags dumpFlags) error {
 	}
 
 	if configPath != "" {
-		if modules.GetModules().BuildType() == modules.BuildOSS {
-			fmt.Printf("Wrote config to file %q. Now you can start the server. Happy Teleporting!\n", configPath)
+		canWriteToDataDir, err := utils.CanUserWriteTo(flags.DataDir)
```

Comment:
This will now fail with not implemented error on Windows. Should we gracefully ignore it below?